src/engine/Match_tainting_mode.ml

```diff
@@ -242,14 +242,20 @@ let range_of_any any =
    * the AST at some point. *)
   match Visitor_AST.range_of_any_opt any with
   | None ->
+      (match any with
       (* IL.any_of_orig will return `G.Anys []` for `NoOrig`, and there is
        * no point in issuing this warning in that case.
        * TODO: Perhaps we should avoid the call to `any_in_ranges` in the
        * first place? *)
-      if any <> G.Anys [] then
-        logger#warning
```

Comment:
I would rather leave it as it was but just change the severity from `warning` to `trace`.